exporters/otlp/metrics/src/main/java/io/opentelemetry/exporter/otlp/metrics/OtlpGrpcMetricExporterBuilder.java

```diff
@@ -27,10 +28,14 @@
   private static final String DEFAULT_ENDPOINT_URL = "http://localhost:4317";
   private static final URI DEFAULT_ENDPOINT = URI.create(DEFAULT_ENDPOINT_URL);
   private static final long DEFAULT_TIMEOUT_SECS = 10;
+  private static final AggregationTemporality DEFAULT_TEMPORALITY =
```

Comment:
> Does it make sense for exporters to have their own defaults?

I think it does. I think temporality is going to be one of the main design dimensions when building an exporter, since there are implications on the backend design.

Also, `preferredTemporality()` is nullable. If an exporter returns null then the [TemporalityUtils#resolveTemporality](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/TemporalityUtils.java#L21-L38) is called to resolve it. So effectively `TemporalityUtils` owns the definition of default.